### PR TITLE
Feat/artists hero

### PR DESCRIPTION
**Pull request by pdabrowsky**
https://app.asana.com/0/1202705605335915/1202705556786278/f

### CODE REVIEW

File: src/components/Hero/Hero.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import { Button } from '../common/Button/Button';
+
+import './Hero.scss';
+
+const Hero = () => {
+  return (
+    <div className="hero">
+      <div className="hero__content">
+        <img
+          src={require('../../assets/img/background.jpg')}
```
Comment (dominikx96) [inline src/components/Hero/Hero.tsx:11]: 👀 
https://create-react-app.dev/docs/adding-images-fonts-and-files

File: src/components/Hero/Hero.tsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+          alt="background"
```
Comment (dominikx96) [inline src/components/Hero/Hero.tsx:12]: It's a background and I don't think we should add an alt text, leave blank: https://help.siteimprove.com/support/solutions/articles/80000863904-accessibility-image-alt-text-best-practices

+ text is not descriptive, please correct this text: https://bighack.org/how-to-write-better-alt-text-descriptions-for-accessibility/

File: src/components/Hero/Hero.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+        <Button
+          type="button"
+          onClick={() => console.log('click')}
+          className="hero__go-back-btn"
+        >
+          <Icon
+            symbol="Arrow"
+            className="hero__go-back-btn__arrow"
+          />
+          <p>Back to Awards</p>
```
Comment (dominikx96) [inline src/components/Hero/Hero.tsx:24]: Use `span`

File: src/components/Hero/Hero.tsx [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+        </Button>
+        <h1 className="hero__header-text">Ivan Blackstock</h1>
+      </div>
+      <img
+        src={require('../../assets/img/man.jpg')}
```
Comment (dominikx96) [inline src/components/Hero/Hero.tsx:29]: 👀

File: src/components/Hero/Hero.tsx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+        alt="background"
```
Comment (dominikx96) [inline src/components/Hero/Hero.tsx:30]: Incorrect alt text

File: src/components/Hero/Hero.scss [hunk 6]
```diff
@@ -0,0 +1,79 @@
...
+  position: absolute;
+  top: 75%;
+  left: 50%;
+  transform: translateX(-50%);
+  width: 240px;
+  @include breakpoints.small-desktop {
+    top: 67%;
+    width: 412px;
+  }
+}
```
Comment (dominikx96) [inline src/components/Hero/Hero.scss:79]: It should be positioned by `bottom: -x px`, not %

Comment (dominikx96) [reply src/components/Hero/Hero.scss:79]: and position absolute should have a wrapper with image and text, not only image

File: src/components/Hero/Hero.scss [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+    font-size: 1.3rem;
+    top: 35px;
+  }
+}
+
+.hero__go-back-btn__arrow {
+  @include breakpoints.small-desktop {
+    margin-right: 8px;
+  }
+}
```
Comment (dominikx96) [inline src/components/Hero/Hero.scss:67]: Please add a basic left/right animation on hover. Create a separate scss file to store keyframes - you should be able to reuse them

File: src/components/Hero/Hero.scss [hunk 8]
```diff
@@ -0,0 +1,79 @@
...
+    display: flex;
+    justify-content: center;
+  }
+}
+
+.hero__background {
+  object-fit: cover;
+  width: 100%;
+  min-width: 600px;
+  max-width: 1600px;
```
Comment (dominikx96) [inline src/components/Hero/Hero.scss:25]: you don't need that

**Review by dominikx96 — CHANGES_REQUESTED** [review]
It doesn't look like on Figma:
Screenshot 2022-08-29 at 15 37 34

File: src/components/Hero/Hero.scss [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+    font-size: 120px;
+  }
+}
+
+.hero__go-back-btn-container {
+  position: absolute;
+  top: 0;
+  left: 50%;
+  transform: translateX(-50%);
+}
```
Comment (dominikx96) [inline src/components/Hero/Hero.scss:46]: Do not use absolute for everything! As I mentioned you should wrap this content with `Container` component and add position relative, that's everything :D

File: src/components/Hero/Hero.scss [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+.hero__go-back-btn {
+  display: flex;
+  align-items: center;
+  margin: 16px 0 0 24px;
+  font-family: variables.$font-secondary;
+  font-weight: variables.$light;
+  font-size: 1.1rem;
+  letter-spacing: -0.04em;
+  text-transform: uppercase;
+  :hover {
```
Comment (dominikx96) [inline src/components/Hero/Hero.scss:57]: You forgot about `&` and that's why it doesn't work

File: src/components/Hero/Hero.scss [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+
+.hero__go-back-btn__arrow {
+  @include breakpoints.small-desktop {
+    margin-right: 8px;
+  }
+}
+
+.hero__man {
+  position: absolute;
+  object-fit: cover;
```
Comment (dominikx96) [inline src/components/Hero/Hero.scss:77]: 👀  please elaborate what's objectFit position

**Review by dominikx96 — CHANGES_REQUESTED** [review]

File: src/components/Hero/Hero.scss (hunk 11 above)
Comment (pdabrowsky) [reply src/components/Hero/Hero.scss:77]: objectFit position property sets how content (img or video) should be resized to fit its container.

File: src/styles/_variables.scss [hunk 12]
```diff
@@ -15,4 +15,5 @@ $font-primary: 'Futura';
 $font-secondary: 'Futura PT';
 
 $light: 300;
+$normal: 400;
 $bold: 500;
```
Comment (dominikx96) [inline src/styles/_variables.scss:19]: It should have a `weight` prefix, it isn't clear to me what the `$light` variable means.

File: src/styles/_animations.scss [hunk 13]
```diff
@@ -0,0 +1,12 @@
...
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(180deg);
+  }
+}
+
+@mixin animate($animation, $duration, $method) {
+  animation: $animation $duration $method;
+}
```
Comment (dominikx96) [inline src/styles/_animations.scss:12]: good job! 💪

**Review by dominikx96 — CHANGES_REQUESTED** [review]
just 2 small improvements, looks good!

File: src/components/Hero/Hero.tsx [hunk 14]
```diff
@@ -0,0 +1,45 @@
+import { Icon } from '../common/Icon/Icon';
+import { Button } from '../common/Button/Button';
+import { Container } from '../common/Container/Container';
+
+import background from '../../assets/img/background.jpg';
+import man from '../../assets/img/man.jpg';
+
+import './Hero.scss';
+
+const Hero = () => {
```
Comment (dominikx96) [inline src/components/Hero/Hero.tsx:10]: Do not hardcode props. Create props & types for:
- background image url
- artist name
- artist image

**Review by dominikx96 — CHANGES_REQUESTED** [review]

**Review by dominikx96 — APPROVED** [review]

**Pull request merged**